htdocs/core/class/commonobject.class.php

```diff
@@ -783,6 +811,27 @@ abstract class CommonObject
 
 	// No constructor as it is an abstract class
 
+	/**
+	 * Provide list of deprecated properties and replacements
+	 *
+	 * @return array<string,string>
+	 */
+	protected function deprecatedProperties()
+	{
+		return array(
+			'alreadypaid' => 'totalpaid',
+			'cond_reglement' => 'cond_reglement_id',
+			'modelpdf' => 'model_pdf',
```

Comment:
This field has been definitely removed from develop branch so we can remove it here too.

Reply:
@eldy The question is : do you want to provide some backwards compatibility or not.
This table is the mapping in case an access to the old value is made.

If you want me to make deprecated fields unsupported, let me know which ones to remove - I am pretty sure that some need to be kept.